install_files/ansible-base/staging-specific.yml

```diff
@@ -6,11 +6,10 @@ dns_server: "8.8.8.8"
 ### Used by the common role ###
 ssh_users: "vagrant"
 
-# TODO Should use ansible to gather this info
-monitor_ip: "10.0.1.3"
-monitor_hostname: "mon-staging"
-app_hostname: "app-staging"
-app_ip: "10.0.1.2"
+monitor_ip: "{{ hostvars['mon-staging'].ansible_all_ipv4_addresses.0 }}"
```

Comment:
>Notice how we’re pulling out the hostname of the first machine of the webservers group. If you are doing this in a template, you could use the Jinja2 ‘#set’ directive to simplify this, or in a playbook, you could also use set_fact:
>
>        set_fact: headnode={{ groups[[‘webservers’][0]] }}
>        debug: msg={{ hostvars[headnode].ansible_eth0.ipv4.address }}
>
> **Notice how we interchanged the bracket syntax for dots – that can be done anywhere.**

Emphasis added; via [Ansible FAQ](http://docs.ansible.com/faq.html#how-do-i-access-a-variable-of-the-first-host-in-a-group).